internal/services/appconfiguration/app_configuration_resource.go

```diff
@@ -665,6 +786,41 @@ func expandAppConfigurationEncryption(input []interface{}) *configurationstores.
 	return result
 }
 
+func expandAppConfigurationReplicas(input []interface{}, configurationStoreName, configurationStoreLocation string) ([]replicas.Replica, error) {
```

Comment:
We should have this return a pointer instead of an empty value so that if an error occurs we're preventing bad/incorrect data from being used.
```suggestion
func expandAppConfigurationReplicas(input []interface{}, configurationStoreName, configurationStoreLocation string) (*[]replicas.Replica, error) {
```

Reply:
fixed, thanks.